examples/Inference/basic_inference/config_chkpt.yaml

```diff
@@ -0,0 +1,18 @@
+name: test_new_api_basic
+entrypoint: >-
+   python3 -m determined.launch.torch_distributed
+   python3 basic_inference_chkpt.py
+
+# NEW: configure multiple slots per trial.
+resources:
+  slots_per_trial: 2
+
+searcher:
+   name: single
+   metric: x
+   max_length: 100
+
+max_restarts: 0
+bind_mounts:
+  - host_path: /tmp
+    container_path: /data
```

Comment:
can you configure your editor to always emit a newline at the end of the last line?  There's a few files with that issue.